src/App.js

```diff
@@ -10,6 +10,7 @@ import Tasks from './tasks';
 
 const App = (props) => {
 
+  // Why is this required?
```

Comment:
This event listener is used to save the data in local Storage once the user leaves the webpage.